README.md

```diff
@@ -1,44 +1,141 @@
+- [Caliptra GitHub GCP Runner Infrastructure](/ci-tools/github-runner/README.md)
+
+## Directory structure
+
+### [api](/api)
+
+Definitions for the Caliptra mailbox and other basic firmware interfaces.
+
+### [builder](/builder)
+
+Tool for building the Caliptra firmware bundle.
+
+### [cfi](/cfi)
+
+Library containing Control-Flow Integrity attack countermeasure implementations.
+
+### [ci-tools](/ci-tools)
+
+Various tools used for Continuous Integration flows.
+
+### [common](/common)
+
+Common code shared across multiple other code modules.
+
+### [coverage](/coverage)
 
-## [fmc](/fmc/)
+Tool for collecting code coverage metrics from Caliptra tests.
+
+### [cpu](/cpu)
+
+Implementations of CPU-specific features such as NMI and Trap handlers.
+
+### [dpe](/dpe)
+
+DICE Protection Environment submodule (reference to its own repository).
+
+### [drivers](/drivers)
+
+A rust library containing drivers for the Caliptra hardware, intended to be
+used by firmware running on Caliptra's RISC-V cpu.
+
+### [error](/error)
+
+Comprehensive list of all Caliptra error codes.
+
+### [fmc](/fmc/)
 
 "First Mutable Code", the code that the boot ROM measures and jumps to after
 validation succeeds.
 
-## [hw-model](/hw-model/)
+### [hw-latest](/hw-latest/)
+
+Reference to the latest stable Caliptra RTL (reference to its own repository).
+
+### [hw-model](/hw-model/)
 
 A high-level testing library for instantiating and manipulating models of the
 hardware. Intended to target multiple backends, including sw-emulator,
 verilator, and (hopefully) a future FPGA implementation.
 
-## [sw-emulator](/sw-emulator/)
+### [image](/image)
+
+.
+
+### [kat](/kat)
+
+Known Answer Tests for all cryptographic operations supported by Caliptra.
+
+### [libcaliptra](/libcaliptra)
+
+C-API library and example code for accessing Caliptra from its external interfaces.
+
+### [lms-types](/lms-types)
+
+Type definitions for the Leighton-Micali Hash-Based Signatures algorithm.
+
+### [registers](/registers)
+
+Register definitions for the caliptra hardware peripherals, generated from the RDL
+files in the rtl-caliptra repo.
+
+### [rom](/rom/dev)
+
+"Read Only Memory", the code that is first executed when the chip is powered on.
+Normally the ROM is part of the silicon.
+
+### [runtime](/runtime)
+
+Caliptra runtime environment that is responsible for all Caliptra functionality
+after the boot process. This module is jumped to from FMC.
+
+### [sw-emulator](/sw-emulator/)
 
 Fast software-based simulation of the Caliptra hardware. This is the fastest and
 easiest way to test changes to firmware, but fidelity may not be perfect.
 
-## [drivers](/drivers/)
+### [systemrdl](/systemrdl)
 
-A rust library containing drivers for the Caliptra hardware, intended to be
-used by firmware running on Caliptra's RISC-V cpu.
+.
+
+### [test](/test)
+
+Test suites for various portions of the Caliptra implementation.
+
+### [test-harness](/test-harness)
```

Comment:
I need help with a good description for this one

Reply:
"A library for building self-contained test firmware binaries. This set's up minimal infrastructure for panic-handling and provides macros for defining test cases in firmware"

Reply:
`set's up`-->`sets up` and add period at the end.